sdk/src/types/block/output/delegation.rs

```diff
@@ -358,8 +360,13 @@ impl DelegationOutput {
     }
 
     // Transition, just without full ValidationContext.
-    pub(crate) fn transition_inner(current_state: &Self, next_state: &Self) -> Result<(), StateTransitionError> {
-        if !(current_state.delegation_id.is_null() && !next_state.delegation_id().is_null()) {
+    pub(crate) fn transition_inner(
+        current_state: &Self,
+        next_state: &Self,
+        context_inputs: &[ContextInput],
+        protocol_parameters: &ProtocolParameters,
+    ) -> Result<(), StateTransitionError> {
+        if !current_state.delegation_id.is_null() || next_state.delegation_id().is_null() {
```

Comment:
Thibault will come for you

Reply:
Clippy came for me already :cry: Added an allow now